### DISCUSSION TITLE
[Proposal] Reserve WikiText markup delimiters '{% foobar %}' for use as 'official escape' syntax

### DISCUSSION THREAD

**Discussion by PotOfCoffee2Go** [TiddlyWiki/TiddlyWiki5#8137] [Ideas]
This discussion is in response to [talk.tiddlywiki post](https://talk.tiddlywiki.org/t/use-foobar-as-official-escape-syntax/8364).

I have been trying to find where to make that change and where to place the tests. Seems would be somewhere in the parsers? Any help with where in TiddlyWiki repo this change and tests would be applied would be greatly appreciated.

Also is there an agreement that '{% %}' should be be the escape syntax?

Please advise.

**Comment by pmario** [comment]
@Jermolene - I think this discussion should be converted into an issue, so we do not forget about it

As I wrote at Talk, I think "escape syntax" is not the right term, because it is used for regexp string escape sequences. But anyway, I think we should document it.

As I wrote at Talk, I think also the "mushrum brackets" `(= =)`, that the formula plugin uses should be included into that docs.

**Comment by Jermolene** [comment]
Moved to issue #8139
